src/components/date-picker-month/date-picker-month.tsx

```diff
@@ -13,6 +13,8 @@ import { dateFromRange, HoverRange, inRange, sameDate } from "../../utils/date";
 import { DateLocaleData } from "../date-picker/utils";
 import { Scale } from "../interfaces";
 
+const DAYS_PER_WEEK = 7;
+const DAYS_MAXIMUM_INDEX = 6;
 @Component({
```

Comment:
Nitpick: can you add a newline here to visually separate const declaration from the component?